### ISSUE TITLE
More corequisities

### ISSUE THREAD

**Issue by pribism** [oskopek/StudyGuide#27]
One of my subjects has note in SIS: 

> Korekvizity : 	{Alespoň jeden z předmětů MB150P14B, MB150P14E.}

 However, the SG says constraint is broken, even though I have enrolled (or completed) MB150P14B. With enrolled MB150P14E the constraint work.

**Comment by oskopek** [comment]
Yes, well.. StudyGuide currently supports only *AND* conditions. Therefore, when you imported the course, the corequisite constraint was set to `MB150P14B AND MB150P14E`. That's why the constraint is broken.

The problem with this is, it is really hard to parse reliably. Someone can write `Alespon jeden` or `Alespon dva` or `MB150P14E or MB150P14B`, ...

This can of course be fixed:

* A new OR corequisite (and prerequisite) constraint needs to be introduced
* The parsing has to be adjusted

The second step is quite hard to do well. If you have any ideas, please submit. As for now, I don't have the time to fix this.
